bg3-modders-multitool/bg3-modders-multitool/Enums/ValueLists/YesNo.cs

```diff
@@ -0,0 +1,11 @@
+﻿/// <summary>
+/// The YesNo
+/// </summary>
+namespace bg3_modders_multitool.Enums.ValueLists
+{
+    public enum YesNo
+    {
+        No,
```

Comment:
I believe I use a boolean converter to handle this, but this is fine

Reply:
> I believe I use a boolean converter to handle this, but this is fine

It looked literally like this. Tbh no clue why they do such a thing.